charter.md

```diff
@@ -0,0 +1,41 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
+
+## Scope
+
+The team works by prioritizing specific web platform features. The minimum requirements for a feature to be in scope are:
+
+- The feature is defined by a specification from the [IETF](https://www.ietf.org/), [Khronos Group](https://www.khronos.org/), [TC39](https://tc39.es/), [W3C](https://www.w3.org/), or [WHATWG](https://whatwg.org/). Other specifications can be considered on a case-by-case basis.
```

Comment:
Specification is an umbrella term which covers many cases. Some of these are probably not acceptable for the work here. Example: A work proposal, or draft in a W3C WICG is not a good work item for the interop team. The goal is not to implement new things. The goal is to improve interoperability on things already implemented. 

```suggestion
- The feature is defined by a specification —on a standard track when the term exists— from the [IETF](https://www.ietf.org/), [Khronos Group](https://www.khronos.org/), [TC39](https://tc39.es/), [W3C](https://www.w3.org/), or [WHATWG](https://whatwg.org/). Other specifications can be considered on a case-by-case basis.  
**Example**: The [W3C Recommendation track](https://www.w3.org/2021/Process-20211102/#rec-track) is acceptable, but not the [W3C Note track](https://www.w3.org/2021/Process-20211102/#note-track)


- The feature is partially implemented in one or multiple browsers.
```

Issue: The term standard and specification means different things for different organizations. Some are super loosely defined. 

IETF is a good example of how difficult it can become… Anyone can propose an RFC as an [Internet Draft](https://www.ietf.org/how/ids/), and not all RFCs switch to the [standard track](https://www.rfc-editor.org/rfc/rfc2026.html#section-4).

[HTTP Semantics](https://datatracker.ietf.org/doc/rfc9110/) is a good candidate on the standard track, but [OAuth 2.0 for Browser-Based Apps](https://datatracker.ietf.org/doc/draft-ietf-oauth-browser-based-apps/) would not be.

**TODO**: Do we need a table for each organizations on what is standard material or not. -> #121

**TODO**: Should there be an implementation requirement? I added a third criteria. The reasons is that interop group is not here to do prototyping or even be a platform for starting new features even on the standard track. It's about improving the interop state of features which needs care and love to not be a painful point for webdevs. A missing webdev feature across the Web is not necessary an interop issue, it's just an idea which needs work in a standard group/interest group and be refined there. And hopefully if well develop it will never have to go through this group. -> #122

Reply:
Thanks for filing those issues!

Reply:
Proposed wording:
```
- The feature is defined by a standards-track specification from the following organizations:
      * [IETF](https://www.ietf.org/) (Standard Track)
      * [Khronos Group](https://www.khronos.org/)
      * [TC39](https://tc39.es/) (Stage 2+)
      * [W3C](https://www.w3.org/) (Recommendation Track)
      * [WHATWG](https://whatwg.org/)
   Other specifications can be considered on a case-by-case basis.
```

Reply:
Following the discussion in https://github.com/web-platform-tests/interop/issues/127#issuecomment-1234523551 I've removed details about how proposals are evaluated from the charter, and instead added it to https://github.com/web-platform-tests/interop/pull/115.

@fantasai I used your wording almost verbatim, since @jgraham indicated Mozilla would like a standards track expectation as well.